composer.json

```diff
@@ -114,6 +114,7 @@
         "symfony/property-info": "^5.4 || ^6.0",
         "symfony/proxy-manager-bridge": "^5.4 || ^6.0",
         "symfony/routing": "^5.4 || ^6.0",
+        "symfony/runtime": "^5.4 || ^6.0",
```

Comment:
Can we move it to the dev requirement? So we won't preinstall it for others?

Reply:
Primarily, we aim to provide it for Sylius-Standard to allow running Sylius on RoadRunner or FrankenPHP.